pkg/controllers/ttl/manager.go

```diff
@@ -169,8 +169,8 @@ func (m *manager) filterPermissionsResource(resources []schema.GroupVersionResou
 }
 
 func (m *manager) reconcile(ctx context.Context, workers int) error {
-	defer m.logger.Info("manager reconciliation done")
-	m.logger.Info("start manager reconciliation")
+	defer m.logger.V(3).Info("manager reconciliation done")
+	m.logger.V(3).Info("beginning reconciliation after configured interval of ", m.interval)
```

Comment:
```suggestion
	m.logger.V(3).Info("beginning reconciliation", "interval", m.interval)
```

Remember those are key/value pairs.